### PR TITLE
Add tests for Change owner and microchip

### PR DESCRIPTION
**Pull request by miriyalanarahari**
1.Add tests to cover change owner functionality
2.Add tests to remove microchip number
3.Add tests to verify duplicate microchip checks on edit dog details

### CODE REVIEW

File: test/acceptance/features/editDogDetailsvalidations.feature [hunk 1]
```diff
@@ -23,6 +23,13 @@ Feature: Edit dog details
         Then I expect that element "form" contains the text "Microchip numbers must be 15 numbers long"
         Then I expect that element "form" contains the text "Microchip numbers can only contain numbers"
 
+    Scenario: Enter microchip number already exists in the database for another dog
+        When I set "345677654366666" to the inputfield "#microchipNumber"
+        And I set "345677654355555" to the inputfield "#microchipNumber2"
+        And I click on the element "button=Save details"
+        Then I expect that element "form" contains the text "The microchip number already exists"
+        And I expect that element "form" contains the text "The microchip number already exists"
```
Comment (jbarnsley10) [inline test/acceptance/features/editDogDetailsvalidations.feature]: Did you mean to duplicate this line (almost) ?

**Review by jbarnsley10 — APPROVED** [review]
Looks good. Just the question of that duplicated line

File: test/acceptance/features/editDogDetailsvalidations.feature (hunk 1 above)
Comment (HariQA-commits) [reply test/acceptance/features/editDogDetailsvalidations.feature]: Removed the duplicate line and added separate tests to check each microchip number field separately.
